operator/apis/loki/v1/lokistack_types.go

```diff
@@ -1078,6 +1086,12 @@ type LokiStackStorageStatus struct {
 	// +optional
 	// +kubebuilder:validation:Optional
 	Schemas []ObjectStorageSchema `json:"schemas,omitempty"`
+	// SchemaStatus conveys any warnings/messages that apply
+	// to the currently applied schemas
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	SchemaStatus string `json:"schemaStatus,omitempty"`
```

Comment:
Having a custom warning condition adds more visibility for the user that in general some action needs to be taken (e.g. conditions are separately visible in the OpenShift Console). However I would keep the `SchemaStatus` in a slightly different form namely integrating this field in the `ObjectStorageSchema` struct, so that we can indicate which schemas can be removed.